cassdegrees/static/js/requirements.js

```diff
@@ -0,0 +1,174 @@
+//! Vue.js based means of adding/removing requirements.
```

Comment:
This needs to be explicitly labeled as for subplans IMO. Is there a reason we are using a different set of scripts for this vs. programs? I don't see a reason why they can't mix and match (i.e have a Major depend on a Minor?)

Reply:
This file no longer exists since it is redundant. I have renamed requirements.html to subplanrules.html.